tutorials/02_installation.md

```diff
@@ -44,15 +44,17 @@ Be sure to replace `<#>` with a number value, such as `1` or `2`, depending on w
 1. Install dependencies
   ```
   sudo apt-add-repository -s "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -c -s) main"
-  sudo apt-get build-dep -y gz-physics<#>-dev
+  sudo apt-get build-dep -y libgz-physics<#>-dev
   ```
-  Be sure to replace `<#>` with a number value, such as `1` or `2`, depending on which version you need.
+  Be sure to replace `<#>` with a number value, such as `5` or `6`, depending on which version you need.
+  On Focal, versions `2`, `3`, and `4`, can be installed by using the `libignition-physics<#>-dev` packages.
```

Comment:
I would say something like from version 6, you should use libgz-physics<#>-dev, for lower version libignition-physics<#>-dev